examples/stargazers/webpack.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
 
   module: {
     loaders: [
-      { test: /\.js$/, exclude: /node_modules|dist/, loader: "babel-loader" },
+      { test: /\.js$/, exclude: /node_modules|dist/, loader: "babel-loader?stage=0" },
```

Comment:
Thanks for the notes here.  I missed this (subtle) difference.  I like being able to do `propTypes` inside of the class definition, too!